src/nvim/ui.c

```diff
@@ -235,6 +235,42 @@ int ui_pum_get_height(void)
   return pum_height;
 }
 
+void ui_pum_get_pos(double *pwidth, double *pheight, double *prow, double *pcol)
+{
+  double w = 0.0, h = 0.0, r = -1.0, c = -1.0;
+  bool found = false;
+  for (size_t i = 1; i < ui_count; i++) {
+    if (!uis[i]->pum_pos) {
+      continue;
+    }
+    if (!found) {
+      w = uis[i]->pum_width;
+      h = uis[i]->pum_height;
+      r = uis[i]->pum_row;
+      c = uis[i]->pum_col;
+      found = true;
+    } else {
+      w = MIN(uis[i]->pum_width, w);
```

Comment:
I don't think this reduction makes sense. We should probably just return the position of the first UI with `pum_pos` set.

Reply:
updated.